viewing_party/party.py

```diff
@@ -0,0 +1,120 @@
+#WAVE 1
+def create_movie(title, genre, rating):
+    if bool(title) == True and bool(genre) == True and bool(rating) == True:
+        movie_details_dict = {"title": title, "genre": genre, "rating": rating}
+        return movie_details_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data 
+
+def watch_movie(user_data, title):   
+    for movie in user_data['watchlist']:
+        if title == movie['title']:
+            user_data['watchlist'].remove(movie)
+            user_data['watched'].append(movie)
+    return user_data
+
+
+
+#WAVE 2
+def get_watched_avg_rating(user_data):
+    if len(user_data['watched']) == 0:
```

Comment:
A more pythonic way to check whether a list is empty would be
```suggestion
    if not user_data['watched']:
```